lib/tasks/javascript.rake

```diff
@@ -181,6 +181,10 @@ task "javascript:update_constants" => :environment do
     export const ADMIN_SEARCH_RESULT_TYPES = #{Admin::SearchController::RESULT_TYPES.to_json};
 
     export const API_KEY_SCOPE_MODES = #{ApiKey.scope_modes.keys.to_json}
+
+    export const FONTS = #{DiscourseFonts.fonts.map { |font| font[:name] }}
+
+    export const DEFAULT_TEXT_SIZES = #{DefaultTextSizeSetting::DEFAULT_TEXT_SIZES}
```

Comment:
I just merged https://github.com/discourse/discourse/commit/0f0a59d7ff1cc49023057bfa8ace77f31bab4cd1 which means we can move these constants into the admin bundle only.